mtz-ajax-interceptor-behavior.html

```diff
@@ -1,13 +1,136 @@
 <link rel="import" href="../polymer/polymer.html">
 
-<!--
-`mtz.AjaxInterceptorBehavior`
+<script>
+  window.mtz = window.mtz || {};
 
-@demo demo/index.html
--->
+  /**
+   * `mtz.AjaxInterceptorBehavior` handles registering declarative interceptors then listening for all registered events
+   *  and calling the interceptors associated with those events.
+   *
+   * @polymerBehavior mtz.AjaxInterceptorBehavior
+   */
+  mtz.AjaxInterceptorBehavior = {
+    properties: {
+      /* The map of event to interceptor Set to listen against */
+      interceptors: {
+        type: Object,
+        value() {
+          return {};
+        }
+      }
+    },
+    attached() {
+      this.addEventListener('register-interceptor', this.__registerInterceptor);
```

Comment:
I'm with Stramel on this one, there is no DOM to render from this element, all of it's functionality it hinged upon those event listeners, and when combined with something like `<iron-ajax auto>` and when the interceptors fire their registration event, deferring to after the next render may be too late.